appveyor.yml

```diff
@@ -1,22 +1,58 @@
 version: 1.0.{build}
+os:
```

Comment:
AppVeyor treats the "os" setting as part of the "test matrix" - i.e. the run will happen for each OS and AppVeyor will provision a machine with Windows and VS 2013 / 2015, but not both